packages/contact-center/store/src/store.types.ts

```diff
@@ -0,0 +1,31 @@
+import {AgentLogin, IContactCenter, Profile, Team} from '@webex/plugin-cc';
+
+interface WithWebex {
+    webex: { cc: IContactCenter };
+}
+
+interface WithWebexConfigAndToken {
+    webexConfig: any; // Replace 'any' with the actual type of webexConfig
+    access_token: string;
+}
+  
+type InitParams = WithWebex | WithWebexConfigAndToken;
+
+interface IStore {
+    teams: Team[];
+    loginOptions: string[];
+    cc: IContactCenter;
+  
+    registerCC(webex: WithWebex['webex']): Promise<Profile>;
+    init(params: InitParams): Promise<void>;
+}
+
+export type {
+    IContactCenter,
+    Profile,
+    Team,
+    AgentLogin,
+    WithWebex,
```

Comment:
We dont need to export this type, its already exported with initParams, right?

Reply:
In store.ts, we have registerCC method which receives `webex` object. Although it is typed as `any` for now, if we update its type later, wanted it to work automatically. So referred the type from `WithWebex` like this:

```javascript
registerCC(webex: WithWebex['webex']): Promise<void> {
    //register code
}
```